code/modules/mod/mod_theme.dm

```diff
@@ -796,15 +778,13 @@
 	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 15, BOMB = 15, RAD = 50, FIRE = INFINITY, ACID = 450) //On one hand this is quite strong, on the other hand energy hole / antagonists need to steal, and thus by extention use this.
 
 /datum/mod_theme/praetorian
-	name = "praetorian"
-	desc = "A prototype of the Magnate-class suit issued to station Blueshields, still boasting exceptional protection worthy of an honor guard."
-	extended_desc = "A prototype of the Magnate-class suit issued for use with the station Blueshields, \
-		it boasts most of the exceptional protection of it's successor, while sacrificing some of the module capacity.\
-		Most of the protection of the Magnate, with none of the comfort! The visor uses blue-light to obscure \
-		the face of it's wearer, adding to it's imposing figure. Compared to the sleek and luxurious design \
-		that came after it, this suit does nothing to hide it's purpose, the reinforced plating layered \
-		over the insulated inner armor granting it protection against corrosive liquids, explosive blasts, \
-		fires, electrical shocks, and contempt from the rest of the crew."
+	name = "'Praetorian' escort"
+	desc = "A Nanotrasen refit of the Takabara modsuit line. Offers higher mobility than the base model, and comes with a snazzy blue-and-black paint job."
+	extended_desc = "A licensed copy of the Shellguard Munitions 'Takabara' modsuit, the NA-35 'Praetorian' is specially issued to Nanotrasen Asset Protection's Blueshield Corps: \
+		elite protective agents charged with guarding Nanotrasen VIPs. As such, the suit's motor and weight distribution systems have seen a measure of improvement, to allow for the \
```

Comment:
"[...] Nanotrasen Asset Protection's Blueshield Corps: \  elite protective agents charged with guarding Nanotrasen VIPs."

Perhaps replace with : "Nanotrasen Asset Protection's Blueshield Corps: \  elite bodyguards charged with the safety of Nanotrasen VIPs"

The only definition of "Protective agent" I could find either refers to someone in shipping who looks after goods, or chemical agents.